src/js/blue-button/config.js

```diff
@@ -0,0 +1,162 @@
+module.exports = {
+  reportTypes: {
+    appointments: [
+      {
+        value: 'futureappointment',
+        label: 'Future VA Appointments',
+      },
+      {
+        value: 'pastappointment',
+        label: 'Past VA Appointments (limited to past 2 years)',
+      },
+    ],
+    medications: [
+      {
+        value: 'prescription',
+        label: 'VA Medication History',
+      },
+      {
+        value: 'medication',
+        label: 'Medications and Supplements, Self-Reported',
+      },
+    ],
+    labs: [
+      {
+        value: 'labsandtest',
+        label: 'VA Laboratory Results',
+        hold: 3,
+      },
+      {
+        value: 'vapatholog',
+        label: 'VA Pathology Reports',
+        hold: 14,
+      },
+      {
+        value: 'varadiolog',
```

Comment:
`varadiology`
Looks like a lot of these are missing the last letter
`welnes` -> `wellness`, etc.